_includes/setup/system-win.md

```diff
@@ -0,0 +1,11 @@
+## System requirements
+
+To install and run Flutter, your development environment must meet these minimum requirements:
+
+* **Operating Systems**: Windows (64-bit)
```

Comment:
For all platforms (not just WIndows), Flutter tools currently assumes that you're running on 64bits. If we want to run on 32bit, we probably could, but we would have to add some logic to download the 32bit version of the Dart SDK (and possibly of our other pre-build binaries) if we detect such a platform. Personally, I don't think we will gain much by also supporting 32bit. If you disagree, you should file a feature request and we can discuss it there :)